src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -321,6 +321,18 @@ public void declarative() throws Exception {
         r.assertLogContains("Apache Maven 3.3.9", b);
     }
 
+    @Issue("JENKINS-46085")
```

Comment:
can we have this test in a new file ContainerLogStepTest.java, this one is getting too big